### DISCUSSION TITLE
Settings configuration file location

### DISCUSSION THREAD

**Discussion by Surfinokie** [microsoft/azuredatastudio#15704] [Q&A]
We are using ADS in an enterprise environment and I would like to customize the settings, in this specific case the "Update" setting so users are not nagged when an update is available. I cannot find the configuration file where these data are stored to customize them at install time for users. Where do those types of settings live?

**Comment by Charles-Gagnon** [comment]
https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations

(replace instances of `Code` with `azuredatastudio` - e.g. `%APPDATA%\azuredatastudio\User\settings.json`)

**Reply by Surfinokie** [reply]
Perfect, many thanks Charles. That showed me what I need.
